Sources/WebService/WebService.swift

```diff
@@ -37,20 +37,20 @@ import Foundation
      
      A trailing forward slash in **not required** at the end of the URL.
     */
-    static var serverURL: String { get set }
+    var serverURL: String { get set }
```

Comment:
@abhishekfk This a massive change for WebService, since it breaks a lot of existing assumptions and syntax that Swifty provides. Is there a reason we want to do this? If the idea is to change these variables during runtime, these variables are still settable while being static, like `HTTPBin.serverURL = “https://mynewurl.com”`

Reply:
With static server and serverURL, dependency injection for WebService become difficult. 
For exmple:
```
Class Test {
    var dependency: Webservice
}

If I want two webservices:

Class OriginalWebservice: Webservice {
  static func something() {
     }
}

Class MockWebservice: Webservice {
   static func something() {
        
     }
}

let objectWithActualService = Test()
objectWithActualService.dependency = OriginalWebservice()
objectWithActualService.dependency.something() //can't do

let objectWithMockService =  Test()
objectWithMockService.dependency =  MockWebservice()
objectWithMockService.dependency.something() //can't do because something is static
```

This is required if we want to write unit tests with Mocks instead of actual server.

Only option left is to inject Class Type instead of object as dependency.

Any reason why WebService protocol has static members?

Reply:
I think we can make these changes later. For now we will use HTTP stubbing for test cases.

Reply:
@abhishekfk Also have you tried using `.mock`?

Reply:
Yes. If we want to test Complete Webservice, then we will have two implementation, one with actual server, one with .mock().
Because Webservice protocol has static methods, switching between actual webservice and mock webservice at run time is not straight forward. That is why I wanted to move away from webservice with static members.
Also currently .mock() doesn't support, HTTP headers, HTTP codes (200, 400) and Error mocking. So These changes should also be picked.

Reply:
Makes sense to me. Let's discuss this